internal/cltest/factories.go

```diff
@@ -270,14 +277,14 @@ func StringToVersionedLogData(str string) hexutil.Bytes {
 	}
 
 	data := hex.EncodeToString(cbor)
-	version := utils.EVMHexNumber(1)
-	offset := "0000000000000000000000000000000000000000000000000000000000000020"
+	version := utils.RemoveHexPrefix(utils.EVMHexNumber(1))
+	offset := "0000000000000000000000000000000000000000000000000000000000000060"
 
 	var endPad string
 	if length%32 != 0 {
 		endPad = strings.Repeat("00", (32 - (length % 32)))
 	}
-	return hexutil.MustDecode(version + offset + lenHex + data + endPad)
+	return hexutil.MustDecode(id + version + offset + lenHex + data + endPad)
```

Comment:
This may be because of my systems programming/C background, but I generally prefer to do things bytewise rather than string wise, what do you say to refactoring this a little bit?

Reply:
I think that setting things like `offset` would be significantly less straightforward if done in binary. This is also a test helper, so I tend to prioritize readability over performance.